src/applications/edu-benefits/feedback-tool/config/form.js

```diff
@@ -142,17 +144,29 @@ function manualSchoolEntryIsCheckedAndIsUS(formData) {
 }
 
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: '/v0/gi_bill_feedbacks',
   submit,
   trackingPrefix,
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
   formId: VA_FORM_IDS.FEEDBACK_TOOL,
+  benefitDescription: {
+    benefitType: 'feedback',
+    subText: 'GI Bill School Feedback Tool',
+  },
   version: 1,
   migrations,
   prefillEnabled: true,
   prefillTransformer,
+  customText: {
+    // Without this, SiP would say "Your saved feedback application (GI Bill
+    // School Feedback Tool) has expired"
+    // With this, the message is "Your saved feedback application (FEEDBACK-TOOL)
+    // has expired"
+    savedFormDescription: 'feedback application (FEEDBACK-TOOL)',
```

Comment:
If it were up to me, I wouldn't have a `savedFormDescription` for this :shrug:

Reply:
How would you ensure each form can have it say what it needs to say in each of the saved form messages?